river/command/set_repeat.zig

```diff
@@ -0,0 +1,40 @@
+// This file is part of river, a dynamic tiling wayland compositor.
+//
+// Copyright 2020 Isaac Freund
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+const std = @import("std");
+
+const c = @import("../c.zig");
+
+const Error = @import("../command.zig").Error;
+const Seat = @import("../Seat.zig");
+
+/// Close the focused view, if any.
+pub fn setRepeat(
+    allocator: *std.mem.Allocator,
+    seat: *Seat,
+    args: []const []const u8,
+    out: *?[]const u8,
+) Error!void {
+    if (args.len < 3) return Error.NotEnoughArguments;
+    if (args.len > 3) return Error.TooManyArguments;
+
+    const rate = try std.fmt.parseInt(i32, args[1], 10);
+    const delay = try std.fmt.parseInt(i32, args[2], 10);
+
+    const keyboard: *c.wlr_keyboard = c.wlr_seat_get_keyboard(seat.wlr_seat);
+    c.wlr_keyboard_set_repeat_info(keyboard, rate, delay);
```

Comment:
It would probably be more correct to iterate over `Seat.keyboards` and set the repeat rate for all keyboards. Ideally we should allow targeting specific keyboards by identifier in the future but that ties into more general input config and we don't yet have the infrastructure for that.